catalog/app/containers/Bucket/PackageDialog/PackageDialog.js

```diff
@@ -190,6 +191,7 @@ export function WorkflowInput({ input, meta, workflowsConfig }) {
       onChange={input.onChange}
       value={input.value}
       disabled={disabled}
+      error={errors[errorKey]}
```

Comment:
```suggestion
      error={errorKey ? (errors[errorKey] || errorKey) : undefined}
```

Reply:
`errors[errorKey] || errorKey` seems enough, as `{}[undefined]` will be `undefined`

Reply:
But this works for me as well

Reply:
> `{}[undefined]` will be `undefined`

well, in theory there may be something under `"undefined"` key (`{ undefined: 1 }[undefined] === 1`)